lib/views/facebook_login_form.jade

```diff
@@ -16,6 +16,8 @@ script(type='text/javascript').
         var queryStr = window.location.search.replace('?', '');
         // TODO make dynamic
         if (queryStr) {
+          // Remove existing access token from query string.
```

Comment:
I agree with you but don't think it's in the scope of this PR :) But I've added the comment and we can refactor this later (there's even a TODO comment to do so ;)).